src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const result = sourceString
+    .split(';')
+    .map((item) => item.split(':'))
+    .map((item) => item.map((value) => value.trim()))
+    .map((item) => item.filter((value) => value !== ''))
+    .filter((item) => item.length !== 0)
+    .reduce((object, item) => {
+      object[item[0]] = item[1];
+
+      return object;
```

Comment:
The parameter `object` in the `reduce` callback can be given a more descriptive name to reflect its purpose, such as `styleObject` or `cssObject`.